models/actions/runner.go

```diff
@@ -153,6 +159,7 @@ type FindRunnerOptions struct {
 	OwnerID       int64
 	Sort          string
 	Filter        string
+	IsOnline      bool
```

Comment:
Maybe `util.OptionalBool` is better. Otherwise you cannot select all the runners.

Reply:
Done.